Sources/SkipWeb/WebEngine.swift

```diff
@@ -221,8 +221,21 @@ public class WebEngineDelegate : android.webkit.WebViewClient {
     }
 
     /// Notify the host application that a page has started loading.
-    override func onPageStarted(view: PlatformWebView, url: String, favicon: android.graphics.Bitmap) {
+    override func onPageStarted(view: PlatformWebView, url: String, favicon: android.graphics.Bitmap?) {
         logger.log("onPageStarted: \(url)")
+        // add support for webkit.messageHandlers.messageHandlerName.postMessage(body)
+        // JS Proxies are pretty weird. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Proxy
+        // We're using an empty target; when JS accesses any property,
+        // we'll return a JS object with a `postMessage` member function, which will call
+        // skipWebAndroidMessageHandler.postMessage, passing the messageHandlerName and body as strings.
+        view.evaluateJavascript("""
```

Comment:
Would it be possible to only execute this script if there are any message handlers? I'm concerned with adding a script execution (regardless to how trivial) to every page load even in the common case when there are no message handlers installed.

Reply:
Done